pallets/collator-assignment/src/lib.rs

```diff
@@ -103,6 +103,7 @@ pub mod pallet {
         type CollatorAssignmentHook: CollatorAssignmentHook<BalanceOf<Self>>;
         type Currency: Currency<Self::AccountId>;
         type CollatorAssignmentTip: CollatorAssignmentTip<BalanceOf<Self>>;
+        type AllowEmptyOrchestrator: Get<bool>;
```

Comment:
@tmpolaczyk what do you think about this?

Reply:
Ugly but works.

Perhaps I would change the variable to mean `DoNotAssignToOrchestrator` or `ForceEmptyOrchestrator`, with the default value of false being the same as before, but when set to true it ignores `min_collators_for_orchestrator` and `max_collators_for_orchestrator` and sets them to 0 instead, and disables the sanity check. To better indicate that not only we allow orchestrator to be empty, we force it to be empty.

Reply:
sounds good

Reply:
@tmpolaczyk done in a1663bdacd529b5057b29aa163b4732dfdd4da76 :)